pkg/ansible/events/log_events.go

```diff
@@ -100,7 +100,7 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 func (l loggingEventHandler) logAnsibleStdOut(e eventapi.JobEvent) {
 	fmt.Printf("\n--------------------------- Ansible Task StdOut -------------------------------\n")
 	if e.Event != eventapi.EventPlaybookOnTaskStart {
-		fmt.Printf(fmt.Sprintf("\n TASK [%v] ******************************** \n", e.EventData["task"]))
+		fmt.Printf("\n TASK [%v] ******************************** \n", e.EventData["task"])
```

Comment:
Wow that `fmt.Sprintf` inside a `fmt.Printf` was weird. Good fix.

Reply:
I did this by a mistake. Passed. If you have the lint would not.